airbyte_cdk/test/entrypoint_wrapper.py

```diff
@@ -301,7 +301,14 @@ def from_completed_process(
         stdout_file: Path | None = None,
     ) -> "EntrypointOutput":
         """Create EntrypointOutput from a completed subprocess with optional stdout file."""
-        instance = cls(message_file=stdout_file)
+        if stdout_file is not None:
+            instance = cls(message_file=stdout_file)
+        elif completed_process.stdout:
+            messages = completed_process.stdout.splitlines()
+            instance = cls(messages=messages)
+        else:
+            instance = cls()
```

Comment:
Let's raise a ValueError here. Either `stdout_file` should be provided, or `completed_process.stdout` should not be `None`.